src/__tests__/index.js

```diff
@@ -10,7 +10,7 @@ describe('40k', () => {
   let gameData
   beforeAll(async () => {
     gameData = await readFiles(path.join(__dirname, 'downloadedGameSystems/wh40k'), fs)
-  })
+  }, 60000)
```

Comment:
Went with a 60 second timeout for test setup. I'm open to lowering it

Reply:
60s should be fine for now.